lib/algolia/search_client.rb

```diff
@@ -673,10 +673,14 @@ def replace_dictionary_entries!(dictionary, dictionary_entries, opts = {})
       # @return DictionaryResponse
       #
       def delete_dictionary_entries(dictionary, object_ids, opts = {})
+        request  = []
+        object_ids.map do |object_id|
+          request.push({ objectID: object_id })
+        end
```

Comment:
Since you're using `map`, you can assign the result to `request` directly.
```suggestion
        request = object_ids.map do |object_id|
          { objectID: object_id }
        end
```